pop-0001.txt

```diff
@@ -0,0 +1,623 @@
+POP: 1
+Title: POP Purpose and Guidelines
+Version: $Revision$
+Last-Modified: $Date$
+Author: Carol Willing
+Status: Active
+Type: Process
+Content-Type: text/x-rst
+Created: 14-Apr-2015
+Post-History: 14-Apr-2015
+
+
+What is a POP?
+==============
+
+POP stands for PyLadies Organizer Proposal.  A POP is a design
+document providing information to the PyLadies community, or describing
+a new feature for PyLadies or its processes or environment.  The POP
+should provide a concise technical specification of the feature and a
+rationale for the feature.
+
+We intend POPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for
+documenting the design decisions that have gone into PyLadies.  The POP
+author is responsible for building consensus within the community and
+documenting dissenting opinions.
+
+Because the POPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+proposal [1]_.
+
+
+POP Types
+=========
+
+There are three kinds of POP:
+
+1. A **Standards Track** POP describes a new feature or implementation
+   for PyLadies. It may also describe an interoperability standard that will
+   be supported outside the standard library for current PyLadies versions
+   before a subsequent POP adds standard library support in a future
+   version.
+
+2. An **Informational** POP describes a PyLadies design issue, or
+   provides general guidelines or information to the PyLadies community,
+   but does not propose a new feature.  Informational POPs do not
+   necessarily represent a PyLadies community consensus or
+   recommendation, so users and implementers are free to ignore
+   Informational POPs or follow their advice.
+
+3. A **Process** POP describes a process surrounding PyLadies, or
+   proposes a change to (or an event in) a process.  Process POPs are
+   like Standards Track POPs but apply to areas other than the PyLadies
+   language itself.  They may propose an implementation, but not to
```

Comment:
`Process POPs are like Standards Track POPs but apply to areas other than the PyLadies language itself.`  Nice evidence of s/python/PyLadies :wink: 

Perhaps that whole line could be deleted.

I imagine a process could be on deploying your location's website, setting up a new chapter, adding new organizers to the organizers' list, etc.